wordpress-mobile/WordPress-iOS
# Issue/7404 settings gravatar site icon

Fixes #7404 

Partial implementation of 7404. This PR has the gravatar and text button displaying in the `Me > My Profile` screen. 

To test:
1. Navigate to `Me > My Profile` and tap on the text button. 
1a. If a profile photo does not exist, the text button will say "Add a Profile Photo" and the default gravatar placeholder image displays.
1b. If the a profile photo is in use the text button will say "Update Profile Photo" and the current profile photo displays.
2. Tapping on the text button presents the gravatar picker: a photo album called "Selfies" appears.
3. Tapping on the profile image also presents the gravatar picker.
4. Changes from the `Me > My Profile` section are reflected in the `Me` screen.
5. If changes are made to the profile photo in the `Me` screen, they are reflected in the `Me > My Profile` screen as well.
